integration/generator/resources/ec2_linux_test_matrix.json

```diff
@@ -2,14 +2,27 @@
   {
     "os": "ubuntu",
     "username": "ubuntu",
+    "instanceType":"t3a.xlarge",
```

Comment:
Why did you choose this instance type?

Reply:
This instance type is the [default one](https://github.com/aws/amazon-cloudwatch-agent/blob/master/integration/terraform/ec2/linux/variables.tf#L1-L4) for Linux AMD.